automatic/git.install/tools/helpers.ps1

```diff
@@ -77,3 +77,11 @@ function Stop-GitSSHAgent()
     Write-Host "Killing any running git ssh-agent instances"
     Get-Process ssh-agent | Where-Object {$_.Path -ilike "*\git\usr\bin\*"} | Stop-Process
 }
+
+function Stop-GitGPGAgent()
+{
+    if (!(Get-Process gpg-agent -ea 0)) { return }
+
+    Write-Host "Killing any running gpg-agent instances"
+    Get-Process gpg-agent | Where-Object {$_.Path -ilike "*\git\usr\bin\*"} | Stop-Process -Force
```

Comment:
This looks like you are trying to only pick up the gpg-agent located in the top `git` directory, unfortunately there is no guarantee that it is called `git` and can be any name that the user chooses.

Additionally, don't use `ilike`, rather use the normal `like` operator as it is case insensitive by default (I am also unsure if `ilike` is PSv2 compatible).

Reply:
Thanks for the feedback. I have a couple follow-up questions before I take another stab at this:

1. You may have noticed the new function is largely similar to the existing `Stop-GitSSHAgent()` function - therefore, the issues raised are also applicable to it. I realize this is outside of the pull request's original scope, but should I fix this while I'm in here, or start a new discussion?
2. To resolve the install path resolution issue, I was planning to use `Get-AppInstallLocation` from `chocolatey-core.extension`. If this fails to resolve a path, is it acceptable to assume no Git for Windows installation exists and skip termination? Or should I consider abandoning a filepath filter altogether, despite the possibility of this unintentionally terminating other installed instances (e.g. GnuPG, Vagrant, etc.)?

Side note: I just spun up a Windows 7 VM, and can confirm `-ilike` is PSv2-compatible. Nonetheless, I will change this per your request.

Reply:
1. Since they can be considered related, and possibly a bug at the same time it is fine if you include the change for that one as well.
2. Yes, in that case it is fine to skip the termination

> Side note: I just spun up a Windows 7 VM, and can confirm -ilike is PSv2-compatible. Nonetheless, I will change this per your request.

👍